mender-convert-package

```diff
@@ -194,7 +194,7 @@ mender_create_artifact "${device_type}" "${artifact_name}"
 
 log_info "Creating Mender compatible disk-image"
 
-sdimg_path=deploy/${image_name}.sdimg
+img_path=deploy/${image_name}.mender.img
```

Comment:
Outputs: `filename.mender.img.gz`, instead of `filename-mender.img.gz` like in the ticket. To me it made more sense. But your call @mirzak

Reply:
I think it should be `-mender.img` as was in the ticket.

Reply:
Done!